relay-tools/relay-graphql-js
# import vscode-apollo-relay git history, move to workspaces

- break `vscode-apollo-relay` out into multiple packages
	- `@relay-graphql-js/validation-rules`
    - `@relay-graphql-js/generate-config`
- retain `vscode-apollo-relay` package name that imports the above
- add a `@relay-graphql-js/graphql-config` package for `graphql-config`, that similarly imports the above
- set up yarn workspaces, linting, tests, typescript project references for monorepos
- add changesets and github actions for CI/CP

git history notes:
- `vscode-apollo-relay/src/generateConfig` history appears to be gone. i originally `git mv`'d it to `packages/generate-config/src` but should have kept it there. there may be a way to rectify this in the PR
- ?
